### PR TITLE
[keymap] Added abnt2 format to dz60

### PR DESCRIPTION
**Pull request by luizcorreia**
Added abnt2 format to dz60

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by luizcorreia** [comment]
Now my keymap have passed on test.

File: keyboards/dz60/dz60.h [hunk 1]
```diff
@@ -278,6 +278,33 @@
     { k40,  k41,   KC_NO, k43,  KC_NO, KC_NO, k46,  KC_NO, KC_NO, KC_NO, k4a,  k4b,  KC_NO, k4d,   k4e   }  \
 }
 
+/* LAYOUT_60_iso_abnt2
```
Comment (noroadsleft) [inline keyboards/dz60/dz60.h]:
```suggestion
/* LAYOUT_60_abnt2
```

File: keyboards/dz60/dz60.h [hunk 2]
```diff
@@ -278,6 +278,33 @@
...
+ * │10   │12 │13 │14 │15 │16 │17 │18 │19 │1a │1b │1c │1d │     │
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐2d  │
+ * │20    │22 │23 │24 │25 │26 │27 │28 │29 │2a │2b │2c │1e │    │
+ * ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────┤
+ * │30  │31 │32 │33 │34 │35 │36 │37 │38 │39 │3a │3b │3c │  3d  │
+ * ├────┼───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬──┴─┬────┤
+ * │40  │41  │43  │46                      │4a  │4b  │4d  │4e  │
+ * └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘
+*/
+#define LAYOUT_60_iso_abnt2( \
```
Comment (noroadsleft) [inline keyboards/dz60/dz60.h]:
```suggestion
#define LAYOUT_60_abnt2( \
```

File: keyboards/dz60/keymaps/iso_br_abnt2/keymap.c [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+   * | Tab | Q | W | E | R | T | Y | U | I | O | P |' `|{ [|Enter|
+   * |------------------------------------------------------.    |
+   * | Fn1  | A | S | D | F | G | H | J | K | L | Ç |^ ~|} ]|    |
+   * |-----------------------------------------------------------|
+   * |Sft |\ || Z | X | C | V | B | N | M |, <|. >|; :|/ ?|Shift |
+   * |-----------------------------------------------------------|
+   * |Ctrl |GUI|Alt  |         Space         |Alt  |GUI|Fn2|Ctrl |
+   * `-----------------------------------------------------------'
+   */
+  [_BL] = LAYOUT_60_iso_abnt2(
```
Comment (noroadsleft) [inline keyboards/dz60/keymaps/iso_br_abnt2/keymap.c]:
```suggestion
  [_BL] = LAYOUT_60_abnt2(
```

File: keyboards/dz60/keymaps/iso_br_abnt2/keymap.c [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    KC_GESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,
+    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    BR_ACUT, BR_LBRC,
+    KC_CAPS,   KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,  BR_CCDL,  BR_TILD, BR_RBRC, KC_ENT,
+    KC_LSFT, BR_BSLS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT, BR_SCLN, BR_SLSH, KC_RSFT,
+    KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT, KC_RGUI, MO(_FL),   KC_RCTL),
+
+
+  [_FL] = LAYOUT_60_iso_abnt2(
```
Comment (noroadsleft) [inline keyboards/dz60/keymaps/iso_br_abnt2/keymap.c]:
```suggestion
  [_FL] = LAYOUT_60_abnt2(
```

File: keyboards/dz60/keymaps/iso_br_abnt2/keymap.c [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_DEL,
+    XXXXXXX, RGB_TOG, RGB_MOD, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+    _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+    KC_LSFT, XXXXXXX, XXXXXXX, XXXXXXX, BL_DEC,  BL_TOGG, BL_INC,  BL_STEP, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+    KC_LCTL, KC_LGUI, KC_LGUI,                            XXXXXXX,                            KC_RALT, KC_RGUI, _______, KC_RCTL),
+
+  [_CL] = LAYOUT_60_iso_abnt2(
```
Comment (noroadsleft) [inline keyboards/dz60/keymaps/iso_br_abnt2/keymap.c]:
```suggestion
  [_CL] = LAYOUT_60_abnt2(
```

File: keyboards/dz60/keymaps/iso_br_abnt2/keymap.c [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+#include "keymap_br_abnt2.h"
+
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+ * _______ can be used in place of KC_TRNS (transparent) *
+ * XXXXXXX can be used in place of KC_NO (No Operation)  *
+ * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+
+#define _BL 0
+#define _FL 1
+#define _CL 2
```
Comment (noroadsleft) [inline keyboards/dz60/keymaps/iso_br_abnt2/keymap.c]: Optional, but recommended: use an `enum` to manage the layers:

```c
enum layer_names {
    _BL,
    _FL,
    _CL,
};
```

This is the same as the set of definitions you have currently once compiled.

File: keyboards/dz60/keymaps/iso_br_abnt2/keymap.c [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  /* ISO 6u layout layer 0 (BR abnt2 Layout shown)
```
Comment (noroadsleft) [inline keyboards/dz60/keymaps/iso_br_abnt2/keymap.c]:
```suggestion
  /* Layer 0 (BR ABNT2 Layout shown)
```

File: keyboards/dz60/keymaps/iso_br_abnt2/readme.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
+# ISO-BR abnt2 layout
```
Comment (noroadsleft) [inline keyboards/dz60/keymaps/iso_br_abnt2/readme.md]:
```suggestion
# BR ABNT2 layout
```

File: keyboards/dz60/keymaps/iso_br_abnt2/readme.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+
+This is an ISO-BR abnt2 keymap for the DZ60 in a standard ISO layout.
```
Comment (noroadsleft) [inline keyboards/dz60/keymaps/iso_br_abnt2/readme.md]:
```suggestion
This is a Brazilian ABNT2 keymap for the DZ60 in a standard layout.
```

File: keyboards/dz60/keymaps/iso_br_abnt2/readme.md [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+
+The top-left sends Escape normally, but becomes a quote key when
+tapped whilst either Fn or GUI are held down.
+
+## Layout Image
+
+![Layout Image](https://upload.wikimedia.org/wikipedia/commons/thumb/2/2c/KB_Portuguese.svg/1200px-KB_Portuguese.svg.png)
```
Comment (noroadsleft) [inline keyboards/dz60/keymaps/iso_br_abnt2/readme.md]: This layout image doesn't match the physical layout (no split on the right Shift).

Comment (luizcorreia) [inline keyboards/dz60/keymaps/iso_br_abnt2/readme.md]:
```suggestion
![Layout Image](http://s2.glbimg.com/hlQMZ_E87B3DB_96rVuO_pY2Yy8=/695x0/s.glbimg.com/po/tt2/f/original/2016/06/21/900px-kb_portuguese_brazilsvg_.png)
```

File: keyboards/dz60/info.json [hunk 11]
```diff
@@ -88,6 +88,10 @@
...
-    }
+    },
+    "LAYOUT_iso_abnt2": {
```
Comment (noroadsleft) [inline keyboards/dz60/info.json]:
```suggestion
    "LAYOUT_60_abnt2": {
```

File: keyboards/dz60/info.json [hunk 12]
```diff
@@ -88,6 +88,10 @@
...
...l":"A", "x":1.75, "y":2}, {"label":"S", "x":2.75, "y":2}, {"label":"D", "x":3.75, "y":2}, {"label":"F", "x":4.75, "y":2}, {"label":"G", "x":5.75, "y":2}, {"label":"H", "x":6.75, "y":2}, {"label":"J", "x":7.75, "y":2}, {"label":"K", "x":8.75, "y":2}, {"label":"L", "x":9.75, "y":2}, {"label":"\u00c7", "x":10.75, "y":2}, {"label":"^", "x":11.75, "y":2}, {"label":"}", "x":12.75, "y":2}, {"label":"Shift", "x":0, "y":3, "w":1.25}, {"label":"|", "x":1.25, "y":3}, {"label":"Z", "x":2.25, "y":3}, {"label":"X", "x":3.25, "y":3}, {"label":"C", "x":4.25, "y":3}, {"label":"V", "x":5.25, "y":3}, {"label":"B", "x":6.25, "y":3}, {"label":"N", "x":7.25, "y":3}, {"label":"M", "x":8.25, "y":3}, {"label":"<", "x":9.25, "y":3}, {"label":">", "x":10.25, "y":3}, {"label":":", "x":11.25, "y":3}, {"label":"?", "x":12.25, "y":3}, {"label":"Shift", "x":13.25, "y":3, "w":1.75}, {"label":"Ctrl", "x":0, "y":4, "w":1.25}, {"label":"Win", "x":1.25, "y":4, "w":1.25}, {"label":"Alt", "x":2.5, "y":4, "w":1.25}, {"x":3.75, "y":4, "w":6.25}, {"label":"AltGr", "x":10, "y":4, "w":1.25}, {"label":"Win", "x":11.25, "y":4, "w":1.25}, {"label":"Menu", "x":12.5, "y":4, "w":1.25}, {"label":"Ctrl", "x":13.75, "y":4, "w":1.25}]
```
Comment (noroadsleft) [inline keyboards/dz60/info.json]:
```suggestion
        "layout": [{"label":"Esc", "x":0, "y":0}, {"label":"!", "x":1, "y":0}, {"label":"@", "x":2, "y":0}, {"label":"#", "x":3, "y":0}, {"label":"$", "x":4, "y":0}, {"label":"%", "x":5, "y":0}, {"label":"\u00a8", "x":6, "y":0}, {"label":"&", "x":7, "y":0}, {"label":"*", "x":8, "y":0}, {"label":"(", "x":9, "y":0}, {"label":")", "x":10, "y":0}, {"label":"_", "x":11, "y":0}, {"label":"+", "x":12, "y":0}, {"label":"Backspace", "x":13, "y":0, "w":2}, {"label":"Tab", "x":0, "y":1, "w":1.5}, {"label":"Q", "x":1.5, "y":1}, {"label":"W", "x":2.5, "y":1}, {"label":"E", "x":3.5, "y":1}, {"label":"R", "x":4.5, "y":1}, {"label":"T", "x":5.5, "y":1}, {"label":"Y", "x":6.5, "y":1}, {"label":"U", "x":7.5, "y":1}, {"label":"I", "x":8.5, "y":1}, {"label":"O", "x":9.5, "y":1}, {"label":"P", "x":10.5, "y":1}, {"label":"`", "x":11.5, "y":1}, {"label":"{", "x":12.5, "y":1}, {"label":"Caps Lock", "x":0, "y":2, "w":1.75}, {"label":"A", "x":1.75, "y":2}, {"label":"S", "x":2.75, "y":2}, {"label":"D", "x":3.75, "y":2}, {"label":"F", "x":4.75, "y":2}, {"label":"G", "x":5.75, "y":2}, {"label":"H", "x":6.75, "y":2}, {"label":"J", "x":7.75, "y":2}, {"label":"K", "x":8.75, "y":2}, {"label":"L", "x":9.75, "y":2}, {"label":"\u00c7", "x":10.75, "y":2}, {"label":"^", "x":11.75, "y":2}, {"label":"}", "x":12.75, "y":2}, {"label":"Enter", "x":13.75, "y":1, "w":1.25, "h":2}, {"label":"Shift", "x":0, "y":3, "w":1.25}, {"label":"|", "x":1.25, "y":3}, {"label":"Z", "x":2.25, "y":3}, {"label":"X", "x":3.25, "y":3}, {"label":"C", "x":4.25, "y":3}, {"label":"V", "x":5.25, "y":3}, {"label":"B", "x":6.25, "y":3}, {"label":"N", "x":7.25, "y":3}, {"label":"M", "x":8.25, "y":3}, {"label":"<", "x":9.25, "y":3}, {"label":">", "x":10.25, "y":3}, {"label":":", "x":11.25, "y":3}, {"label":"?", "x":12.25, "y":3}, {"label":"Shift", "x":13.25, "y":3, "w":1.75}, {"label":"Ctrl", "x":0, "y":4, "w":1.25}, {"label":"Win", "x":1.25, "y":4, "w":1.25}, {"label":"Alt", "x":2.5, "y":4, "w":1.25}, {"x":3.75, "y":4, "w":6.25}, {"label":"AltGr", "x":10, "y":4, "w":1.25}, {"label":"Win", "x":11.25, "y":4, "w":1.25}, {"label":"Menu", "x":12.5, "y":4, "w":1.25}, {"label":"Ctrl", "x":13.75, "y":4, "w":1.25}]
```

**Comment by luizcorreia** [comment]
Changed! Should be ready for merge ?

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**